src/vs/vscode.d.ts

```diff
@@ -6221,11 +6221,16 @@ declare module 'vscode' {
 		export function createInputBox(): InputBox;
 
 		/**
-		 * Create a new [output channel](#OutputChannel) with the given name.
+		 * Creates a new [output channel](#OutputChannel) with the given name.
+		 * By default the UI will not show the appended data immediately.
+		 * To force UI to show the data immediately pass options with `force` parameter set to `true`.
+		 *
+		 * *NOTE* Default output channels help in improving the performance of VS Code.
 		 *
 		 * @param name Human-readable string which will be used to represent the channel in the UI.
+		 * @param options Optional options to control how the channel will be created.
 		 */
-		export function createOutputChannel(name: string): OutputChannel;
+		export function createOutputChannel(name: string, options?: { force?: boolean }): OutputChannel;
```

Comment:
why an object? are there plans for more options?

Reply:
Yeah, there is a request to set a lang id for channels. If so it has to be set while creating